tutorials/hello-kubernetes/README.md

```diff
@@ -94,6 +94,8 @@ component.dapr.io/statestore created
 
 ## Step 3 - Deploy the Node.js app with the Dapr sidecar
 
+If you want to see the API logs, add `dapr.io/enable-api-logging: "true"` annotation to node.yaml file.
```

Comment:
Instead, add this by default to the applications and change the wording to instruct users to check the logs too. The goal is to demonstrate the feature while also automated the testing of it.